src/client/SConscript

```diff
@@ -14,6 +14,7 @@ def scons():
         SConscript('dfs/SConscript')
         SConscript('dfuse/SConscript')
         SConscript('pydaos/SConscript')
+	SConscript('serialize/SConscript')
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 32)